bin/templates/cordova/lib/builders/GradleBuilder.js

```diff
@@ -211,3 +209,46 @@ module.exports = GradleBuilder;
 function isAutoGenerated(file) {
     return fs.existsSync(file) && fs.readFileSync(file, 'utf8').indexOf(MARKER) > 0;
 }
+
+/**
+ * A special superspawn-like implementation, required to workaround the issue
+ *   with java printing some necessary information to stderr instead of stdout.
+ *   This function redirects all stderr messages to current process stdout. See
+ *   https://issues.apache.org/jira/browse/CB-9971 for explanation.
+ *
+ * @param   {String}    cmd   A command to spawn
+ * @param   {String[]}  args  Command arguments. Note that on Windows arguments
+ *   will be concatenated into string and passed to 'cmd.exe' along with '/s'
+ *   and '/c' swithces for proper space-in-path handling
+ *
+ * @return  {Promise}        A promise, rejected with error message, if
+ *   underlying command exits with nonzero exit code, fulfilled otherwise
+ */
+function spawnWithStderrRedirect(cmd, args) {
+    return Q.Promise(function (resolve, reject) {
+        // Work around spawn not being able to find .bat files.
+        var opts = { stdio: [0,1,1] };
+        if (process.platform === 'win32') {
+            var joinedArgs = [cmd]
+                .concat(args)
+                .map(function(a){
+                    if (/^[^"].* .*[^"]/.test(a)) return '"' + a + '"';
```

Comment:
Fair enough. However if `superspawn` code has a bug and is updated, then this code will be out of date. It's better not to duplicate code and instead modify and reuse `superspawn`.